test/integration/basic/basic.go

```diff
@@ -0,0 +1,180 @@
+package basic
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/test/cluster"
+	"gotest.tools/assert"
+)
+
+type BasicTestRunner struct {
+	cluster.ClusterTestRunnerBase
+}
+
+func (r *BasicTestRunner) RunTests(ctx context.Context) {
+
+	tick := time.Tick(5 * time.Second)
+	wait_for_conn := func(cc *cluster.ClusterContext) {
+		timeout := time.After(120 * time.Second)
+		for {
+			select {
+			case <-timeout:
+				assert.Assert(r.T, false, "Timeout waiting for connection")
+			case <-tick:
+				vir, err := cc.VanClient.VanRouterInspect(ctx)
+				if err == nil && vir.Status.ConnectedSites.Total == 1 {
+					r.T.Logf("Van sites connected!\n")
+					return
+				} else {
+					fmt.Printf("Connection not ready yet, current pods state: \n")
+					r.Pub1Cluster.KubectlExec("get pods -o wide")
+				}
+			}
+		}
+	}
+	wait_for_conn(r.Pub1Cluster)
+	wait_for_conn(r.Priv1Cluster)
+}
+
+func (r *BasicTestRunner) Setup(ctx context.Context, createOptsPublic types.VanSiteConfig, createOptsPrivate types.VanSiteConfig) {
+	var err error
+	err = r.Pub1Cluster.CreateNamespace()
+	assert.Assert(r.T, err)
+
+	err = r.Priv1Cluster.CreateNamespace()
+	assert.Assert(r.T, err)
+
+	createOptsPublic.Spec.SkupperNamespace = r.Pub1Cluster.CurrentNamespace
+	r.Pub1Cluster.VanClient.VanRouterCreate(ctx, createOptsPublic)
```

Comment:
Suggest adding assert on err returned from VanRouterCreate

(and below...)

Reply:
good catch!